unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Camera/CameraController.cs

```diff
@@ -26,8 +26,7 @@ public class CameraController : MonoBehaviour
 
     private void Start()
     {
-        if (RenderingController.i != null)
-            RenderingController.i.OnRenderingStateChanged += OnRenderingStateChanged;
+        CommonScriptableObjects.rendererState.OnChange += OnRenderingStateChanged;
```

Comment:
We shouldn't call `OnRenderingStateChanged` for a first time here? To account for if rendering state was changed before `Start()` call. This also applies to all components/classes that are suscribing to `OnChange` upon initialization.